modules/nf-core/gecco/convert/main.nf

```diff
@@ -0,0 +1,57 @@
+process GECCO_CONVERT {
+    tag "$meta.id"
+    label 'process_single'
+
+    conda "${moduleDir}/environment.yml"
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gecco:0.9.10--pyhdfd78af_0':
+        'biocontainers/gecco:0.9.10--pyhdfd78af_0' }"
+
+    input:
+    tuple val(meta ), path(clusters)
+    tuple val(meta2), path(gbk)
```

Comment:
Why these two inputs here? According with the documentation of the tool, you provide mode (either gbk or clusters) and the path to the input directory containing files to convert, so these two lines could be jut one line like:
`tuple val(meta ), path(input_dir)`